src/main/java/net/helix/pendulum/service/milestone/impl/MilestoneTrackerImpl.java

```diff
@@ -306,8 +309,9 @@ public boolean processMilestoneCandidate(TransactionViewModel transaction) throw
                         // - there doesn't already exist a milestone with the same address for that round
 
                         long calculatedRoundIndex = getRound(transaction.getAttachmentTimestamp());
+                        tracer.trace("ri, calculated ri= {}, {}", roundIndex, calculatedRoundIndex);
                         if (roundIndex == calculatedRoundIndex && isRoundActive(transaction.getAttachmentTimestamp())) {
-
+                            tracer.trace("round is active");
```

Comment:
Sidenote: we should define "active" and "not active"
`isRoundActive`:  getRound based on attachmentTimestamp and see if this round is currently in "active" interval or "pause" interval.